packages/react-components/components/SettingsItem.tsx

```diff
@@ -53,6 +53,21 @@ export function SettingsItemTextValue({
   )
 }
 
+type SettingsItemTextProps = BaseProps
```

Comment:
I think we can refactor `<SettingsItemTextProps>` above and add something like `showChevron` there instead of creating a new component? Wdyt?